README.mkd

```diff
@@ -229,6 +229,41 @@ registers and then save `ra` and the appropriate number of registers from
 epilogue. They restore the saved registers, deallocate the stack space for the
 register, and then perform a return through the restored value of `ra`.
 
+## Conventions for vendor extension
+
+Custom instruction is an important part of RISC-V, offical ISA spec has reserved
```

Comment:
"Support for custom instruction set extensions are an important part of RISC-V, with large encoding spaces reserved of vendor extensions."